MapboxNavigation/PollyVoiceController.swift

```diff
@@ -158,8 +165,15 @@ public class PollyVoiceController: RouteVoiceController {
                     
                     if let audioPlayer = strongSelf.audioPlayer {
                         try strongSelf.duckAudio()
-                        audioPlayer.volume = strongSelf.volume
-                        audioPlayer.play()
+                        
+                        let readyToyPlay = audioPlayer.prepareToPlay()
```

Comment:
Nit: typo.
Also, I would assume `prepareToPlay()` should be called on a background thread but initializing an `AVAudioPlayer` in a background thread is what caused the jittering last time IIRC?
Either way, I think 
```swift
…   try AVAudioPlayer(data: data)
…  .prepareToPlay()
```
should be called before we dispatch to the main thread.